server/init.js

```diff
@@ -0,0 +1,13 @@
+const Routes = require('./routes');
+const React = require('react');
+const ReactDOM = require('react-dom');
+const { Router } = require('react-router');
+
+if (typeof window === 'object') {
```

Comment:
I'll also put the script tag at the bottom of the body so I can remove the event listener.